src/components/capture-area/CaptureAreaSearchValue/CaptureAreaSearchValue.vue

```diff
@@ -39,7 +39,7 @@
                 color="error"
                 prepend-icon="mdi-delete"
                 rounded="lg"
-                @click="vigad.deleteCaptureArea(captureAreaId)"
+                @click="vigad.deleteCaptureArea(captureAreaId);"
```

Comment:
```suggestion
                @click="vigad.deleteCaptureArea(captureAreaId)"
```
We dont need that semicolon there